shadows/framework/src/main/java/org/robolectric/shadows/ShadowContentResolver.java

```diff
@@ -825,8 +831,10 @@ public void setNextDatabaseIdForInserts(int nextId) {
    * DeleteStatement}s invoked on this {@link ContentResolver}.
    *
    * @return a list of statements
-   * @deprecated This method does not work with {@link
-   *     android.content.ContentResolver#acquireContentProviderClient}
+   * @deprecated This method affects all calls, and does not work with {@link
+   *     android.content.ContentResolver#acquireContentProviderClient}. Instead, use
+   *    {@link org.robolectric.Robolectric#setupContentProvider(Class, String)} to
+   *    install a test-specific ContentProvider that can return any Cursor.
```

Comment:
s/that can return any Cursor/that has logic to keep a record of statements

Reply:
It's done in the latest pushed changes, please review